src/api/assets/asset-types.md

```diff
@@ -43,6 +43,7 @@ applicable. When blank, the Asset is not managed by a Centrapay Account.
  * 🅃  : **Test Asset** -- Cannot be assigned to [Merchants] without the test flag.
  * 🚫 : **Restricted Asset** -- Cannot be used to pay for "restricted" [Line Items][].
  * 💸 : **Supports Quick Pay** -- Can be used to quick pay a [Payment Request][].
+ * 💼 : **Requires Tax Number** -- [Tax Number][] is required to create a Payment Request for a Merchant under a Business.
```

Comment:
Could this be reworded to something like `...must exist on the Merchant's Business to transact this asset type`

Reply:
Approved to unblock :)

Reply:
Yep, agreed thanks